fineract-provider/src/main/java/org/apache/fineract/infrastructure/security/filter/TenantAwareTenantIdentifierFilter.java

```diff
@@ -97,7 +97,7 @@ public void doFilter(final ServletRequest req, final ServletResponse res, final
 
             // allows for Cross-Origin
             // Requests (CORs) to be performed against the platform API.
-            response.setHeader("Access-Control-Allow-Origin", "*");
+            response.setHeader("Access-Control-Allow-Origin", "Fineract-Platform-TenantId");
```

Comment:
@vorburger  When I ran this check locally, the build passed all tests. Can you let me know what you are looking for specifically?

Reply:
The files updated were related PR 1091. After the fact,  the same updates were requested on the develop branch. The failing checks were independent of this PR.

Reply:
The star must be replaced by a string and I entered what seemed most logical in the class.

Reply:
>  entered what seemed most logical in the class.

I don't think this hard-coded String (`Fineract-Platform-TenantId`) makes any sense at all, and I suspect that this would break UI clients.